go/flags/endtoend/vtctld.txt

```diff
@@ -86,7 +86,6 @@ Usage of vtctld:
       --log_err_stacks                                                   log stack traces for errors
       --log_rotate_max_size uint                                         size in bytes at which logs are rotated (glog.MaxSize) (default 1887436800)
       --logtostderr                                                      log to standard error instead of files
-      --master_connect_retry duration                                    Deprecated, use -replication_connect_retry (default 10s)
```

Comment:
We have these flags on vtctld, vtctldclient etc. because we haven't migrated the `mysqld` package to `pflag` yet.